blockchain/genesis_test.go

```diff
@@ -169,3 +329,38 @@ func TestSetupGenesis(t *testing.T) {
 		}
 	}
 }
+
+func genCypressGenesisBlock() *Genesis {
+	copyOfCypressChainConfig := *params.CypressChainConfig
+	genesis := DefaultGenesisBlock()
+	genesis.Config = &copyOfCypressChainConfig
+	genesis.Governance = SetGenesisGovernance(genesis)
+	InitDeriveSha(genesis.Config.DeriveShaImpl)
+	return genesis
+}
+
+func genBaobabGenesisBlock() *Genesis {
+	copyOfBaobabChainConfig := *params.BaobabChainConfig
+	genesis := DefaultBaobabGenesisBlock()
+	genesis.Config = &copyOfBaobabChainConfig
+	genesis.Governance = SetGenesisGovernance(genesis)
+	InitDeriveSha(genesis.Config.DeriveShaImpl)
+	return genesis
+}
+
+func genCustomGenesisBlock(customChainId uint64) *Genesis {
+	genesis := &Genesis{
+		Config: &params.ChainConfig{
+			ChainID:                 new(big.Int).SetUint64(customChainId),
+			IstanbulCompatibleBlock: big.NewInt(2),
+			DeriveShaImpl:           2,
+		},
+		Alloc: GenesisAlloc{
+			{1}: {Balance: big.NewInt(1), Storage: map[common.Hash]common.Hash{{1}: {1}}},
```

Comment:
Isn't the key `common.Address`? It looks not an address. How about changing this?

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1017/commits/93f41f7a96224bbb924a3420e3579ab1c040f50c.

Reply:
Thanks for updating. It is minor but, I think it would be more readable with `common.HexToAddress("0xc0ea08a2d404d3172d2add29a45be56da40e2949")` instead of
`common.BytesToAddress([]byte{1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0})`.

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1017/commits/1453b9034def8313b07d5030912496f1aea72583.